lib/oidcstrategy.js

```diff
@@ -752,4 +577,494 @@ Strategy.prototype._shouldLoadUserProfile = function shouldLoadUserProfile(issue
   return done(null, !skip);
 };
 
+/**
+ * validate id_token, and pass the validated claims and the payload to callback
+ * if code (resp. access_token) is provided, we will validate the c_hash (resp at_hash) as well
+ *
+ * @param {String} id_token
+ * @param {Object} req
+ * @param {Function} callback
+ * @param {String} code (if you want to validate c_hash)
+ * @param {String} access_token (if you want o validate at_hash)
```

Comment:
These should be in the same order as the function prototype.

TYPO on access_token (if you want o -> is you want to)